### DISCUSSION TITLE
Knowledge Adventure Club v1

### DISCUSSION THREAD

**Discussion by gustavofsantos** [Send4BR/community#4] [Knowledge Adventure Club]
Você foi desafiado(a)

![ThatSoundsLikeAChallengeWilliamZabkaGIF](https://user-images.githubusercontent.com/8608517/131670945-e5fdd698-2cb8-417e-870f-2cd140e08e42.gif)

Já tentou aprender alguma coisa e ficou travada? Já tentou aprender alguma coisa e veio outras infinitas igualmente interessantes e acabou que aprendeu nada? Já enfrentou a procrastinação cara a cara e perdeu?

É difícil, as vezes vem um sentimento ruim de frustração e parece que o tempo passa e você não sai do lugar. Mas há uma chance!

Acredito muito no poder do aprendizado em grupo e acho que temos o ambiente perfeito para que as pessoas possam crescer juntas, independente de seus objetivos profissionais. Com foco e um objetivo claro e concreto, você vai conseguir!

Ao participar do Knowledge Adventure Club - KAC, **você irá traçar um plano de melhoria.**

Você vai traçar o seu plano de curto prazo. Você sabe quais são os seus interesses e você será altamente encorajado(a) a tomar responsabilidade sobre a sua carreira.

Querendo ou não, é a sua carreira.

O seu plano pode ser:

- o desenvolvimento de uma aplicação
- construção de uma biblioteca
- aprender uma nova linguagem e publicar uma série de blog posts
- o aprendizado em mexer em alguma ferramenta
- o aprendizado de técnicas de refatoração

Qualquer coisa **concreta** que te faça ir do teu nível de conhecimento atual, para o próximo nível.

Ainda não sabe o que quer aprender ou não sabe como estruturar o seu plano? Isso não é um problema, eu vou te ajudar com isso. O que você precisa agora é de vontade!

O KAC é um grupo de estudos com 6 semanas de duração. Você e mais 7 pessoas vão aprender o que for do seu interesse, compartilhar avanços, ajudar uns aos outros e, o mais importante: depois de 6 semanas, terão mais uma ou outra ferramenta nova no seu cinto de ferramentas.

Eu vou ser o guia. Aquela pessoa que organiza as coisas, conta o tempo, marca os encontros, etc. A pessoa que vai garantir que o KAC seja um local seguro para você aprender, cometer erros e fazer aquelas perguntas que você acha que são *nonsense*, mas que quase nunca são.

**Você será a pessoa que irá fazer o trabalho pesado de aprender.**

Parece interessante?

Bora se inscrever então!

### Vamos as regras

- Grupos de no máximo 8 participantes
- Serão 6 encontros de 1hr, uma vez por semana (o horário será acordado entre os participantes)
- A cada semana os participantes terão uma tarefa em grupo, como ler um artigo ou assistir uma palestra
- A cada semana os participantes terão uma tarefa individual relacionada ao tópico de estudo
- Na última semana, é esperado que você termine o que começou e apresente pra galera
- Tudo bem não poder participar da reunião um dia ou outro, mas você deverá mostrar os seus avanços
- Caso haja mais de 8 pessoas inscritas, as pessoas serão separadas em grupos de tamanho igual, respeitando a capacidade máxima

### Como posso me inscrever?

Fácil e rápido: **reaja com um joinha (👍) nesse post.**

Outros emojis serão desconsiderados, então você pode usar eles a vontade.

Quinta-feira (2/set/21) eu vou voltar nesse post pra pegar a lista de pessoas que deram joinha e selecionar a galera que quiser participar, usando os critérios de seleção caso haja mais de 8 inscritos, e já disparar os e-mails de boas vindas com demais instruções.

**Comment by thaisnep** [comment]
Knowledge Adventure Club (KAC) - Grupo 1: 9 pessoas e um aviãozinho pra contrariar... rsrsrsr

**Comment by gustavofsantos** [comment]
Gurizada, encerrei as inscrições. As pessoas confirmadas são:

- @thaisnep 
- @abnerpersio 
- @tcovalesky 
- @JohnKGS 
- @Fernanda-Kipper 
- @fej0se 
- @letbueno 
- @AndreFilippe 
- @GabriellaHuzyk 
- @rodrigosta 
- @igornfaustino 
- @johnkmedeiros 

Vocês serão divididos em dois grupos. Ainda hoje vocês vão receber um e-mail com instruções com os próximos passos

Thanks!
